### PR TITLE
Upgrade third party deps

### PR DESCRIPTION
**Pull request by jjudd**

### CODE REVIEW

File: rules/scala/workspace.bzl [hunk 1]
```diff
@@ -33,7 +33,6 @@ def scala_artifacts():
         "org.scala-sbt:zinc-compile-core_2.13:{}".format(zinc_version),
         "org.scala-sbt:zinc-core_2.13:{}".format(zinc_version),
         "org.scala-sbt:zinc-persist_2.13:{}".format(zinc_version),
-        "org.scala-sbt:compiler-interface:{}".format(zinc_version),
```
Comment (jjudd) [inline rules/scala/workspace.bzl:36]: This was in here twice.

File: rules/scala/workspace.bzl [hunk 2]
```diff
@@ -18,7 +18,6 @@ scalapb_version = "0.11.17"
 def scala_artifacts():
     return [
         "ch.epfl.scala:bloop-frontend_2.12:1.0.0",
-        "com.lihaoyi:sourcecode_2.13:0.2.7,",
```
Comment (jadenPete) [inline rules/scala/workspace.bzl:20]: Nice!

File: tests/workspace.bzl [hunk 3]
```diff
@@ -1,19 +1,24 @@
 load("@rules_jvm_external//:defs.bzl", "maven_install")
+load("@rules_scala_annex//rules/scala:workspace.bzl", "zinc_version")
+
+specs2_version = "4.20.9"
+scalatest_version = "3.2.19"
+scalapb_version = "0.11.17"
```
Comment (jadenPete) [inline tests/workspace.bzl]: Nit: Could we import this from `@rules_scala_annex//rules/scala:workspace.bzl`?

**Review by jadenPete — APPROVED** [review]

File: tests/workspace.bzl (hunk 3 above)
Comment (jjudd) [reply tests/workspace.bzl]: Yep. Good catch.

**Pull request merged**